src/fiber/fiber.mli

```diff
@@ -142,6 +142,10 @@ val collect_errors :
     fails or succeeds. *)
 val finalize : (unit -> 'a t) -> finally:(unit -> unit t) -> 'a t
 
+(** [reraise_all exns] re-raises all [exns] in parallel to the current error
```

Comment:
Maybe change the word order: "to the current error handler in parallel"? (to avoid parsing as "parallel to")

(actually, I don't know why we need to say "in parallel" at all, you can't raise multiple exceptions in sequence, can you? but OK)